codegen/src/sbt-test/generator/e2e/src/test/scala/proto/test/MultiServiceSpec.scala

```diff
@@ -0,0 +1,32 @@
+package proto.test
+
+import com.twitter.util.{Await, Future}
+import com.soundcloud.twinagle.ServerBuilder
+
+import org.specs2.mutable.Specification
+
+class MultiServiceSpec extends Specification {
+  
+  val svc1 = new Service1Service {
+    override def rpc1(req: Request): Future[Response] = Future.value(Response())
+  }
+  
+  val svc2 = new Service2Service {
+    override def rpc2(req: Request): Future[Response] = Future.value(Response())
+  }
+
+
+
+  "ServerBuilder allows building services that support multiple Protobuf services" in {
+    val httpService = ServerBuilder()
+      .register(svc1)
+      .register(svc2)
+      .build
```

Comment:
This "ship has probably sailed", but it wouldn't bother me if this were the only way of building a server (i.e. no convenience `.server` factory method).